app/components/loading/index.tsx

```diff
@@ -8,28 +8,24 @@ import {StyleSheet, View, ViewStyle} from 'react-native';
 type LoadingProps = {
     containerStyle?: ViewStyle;
     style?: ViewStyle;
+    color?: string;
 }
 
-const Loading = ({containerStyle, style}: LoadingProps) => {
+const Loading = ({containerStyle, style, color}: LoadingProps) => {
     return (
-        <View style={[styles.container, containerStyle]}>
+        <View style={containerStyle}>
             <LottieView
                 source={require('./spinner.json')}
                 autoPlay={true}
                 loop={true}
                 style={[styles.lottie, style]}
+                colorFilters={color ? [{color, keypath: 'Shape Layer 1'}, {color, keypath: 'Shape Layer 2'}] : []}
             />
         </View>
     );
 };
 
 const styles = StyleSheet.create({
-    container: {
-        flex: 1,
-        justifyContent: 'center',
-        padding: 20,
-        maxHeight: 40,
```

Comment:
Some of these properties seemed that made no sense, and were interfering in other components, so I left the container styles completely to the caller.

Reply:
I tried to look and test wherever possible the other components, and seemed OK. Nevertheless, I may have missed something, and some minor regression may have appeared, but nothing breaking, and nothing we cannot find during the alpha testing stage.